pages/extensions/amp_dev/markdown_extras/block_filter.py

```diff
@@ -41,3 +43,35 @@ def _get_attributes(match):
     for match in matches:
         attributes[match[0]] = match[1]
     return attributes
+
+FILTERED_SECTION_PATTERN = re.compile(r'({% call filter\(.*?\) %})(.*?)({% endcall %})', re.MULTILINE | re.DOTALL)
```

Comment:
Is this the same pattern used by the normal filter? Does it make sense to extract this?

Reply:
Not quite. That's the step in between: we have `[filter *]` in our Markdown which get's rewritten to jinja2 calls like the one in the pattern (`{% call filter * %}`) which then get evaluated to SSR statements. The right thing to do (which would als simplify all of this here) is to skip jinja2. Created https://github.com/ampproject/amp.dev/issues/3657 to track this.

I held back from doing this directly as I haven't thought about any implications this might have.

Reply:
great - thanks!